server/app/attestation-service-ctl/src/management.rs

```diff
@@ -0,0 +1,150 @@
+use crate::management_api::Tee;
+use anyhow::{anyhow, Context, Result};
+use std::fs;
+use std::io::prelude::*;
+use std::path::Path;
+
+use crate::management_api::management_service_client::ManagementServiceClient;
+use crate::management_api::{
+    GetPolicyRequest, GetPolicyResponse, GetReferenceDataRequest, GetReferenceDataResponse,
+    RestoreDefaultPolicyRequest, RestoreDefaultReferenceDataRequest, SetPolicyRequest,
+    SetReferenceDataRequest,
+};
+
+const DEFAULT_MANAGEMENT_SOCKET: &str = "127.0.0.1:3001";
+
+impl Tee {
+    fn from_str(tee: &str) -> Result<Tee> {
+        match tee {
+            "sgx" => Ok(Tee::Sgx),
+            "tdx" => Ok(Tee::Tdx),
+            "sev-snp" => Ok(Tee::SevSnp),
```

Comment:
The [server sid](https://github.com/confidential-containers/attestation-service/blob/main/server/src/management.rs#L43)e has it named as `sevsnp`, so I guess we should keep it same at both the places. WDYT? Also if updated here, readme would probably need editing